docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -9,6 +9,17 @@ These steps vary based on your current version:
 * <<prepare-to-upgrade,Upgrade from 7.x>>
 endif::[]
 
+[NOTE]
+====
+Elastic maintains several minor versions of the Elastic Stack at once, so
+releases do not always happen in order of their version numbers. You can only
+upgrade to {version} if the version you are currently running has an older
+version number than {version} _and_ has an earlier release date than {version}.
+
+If you are currently running a version with an older version number but a later
+release date than {version}, wait for a newer release before upgrading.
+====
```

Comment:
```suggestion
[IMPORTANT]
====
Elastic maintains several minor versions of the Elastic Stack at once. This means
releases do not always happen in order of their version numbers. You can only
upgrade to {version} if the version you are currently running meets both these
conditions:

* Has an older version number than {version}
* Has an earlier release date than {version}

If you are currently running a version with an older version number but a later
release date than {version}, wait for a newer release before upgrading.
====
```

Reply:
Couple minor optional tweaks for readability + stronger admonition type

Reply:
Maybe "Upgrades to chronologically-older versions will be blocked" is helpful info too?

Reply:
++ thanks, updated & tweaked a little further